docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
<p>Should this be "Open Sourcing Code" subtitle can be the current text. (not a blocker just an idea).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>